### DISCUSSION TITLE
Creatives marketplace

### DISCUSSION THREAD

**Discussion by bartektrybala** [amzn/ads-advanced-tools-docs#90] [General]
Hi, I have a question about creatives' marketplace. AFAIK I can create order for MX market for example, so why I can't do the same for creatives (US is the only option)? Doesn't make much sense.

**Comment by elizafarley** [comment]
Hi @bartektrybala unfortunately this is a known limitation for the DSP creative APIs. The product team is working on fixing this in the next version, and once that functionality is available, we will update the [release notes](https://advertising.amazon.com/API/docs/en-us/info/release-notes).

**Reply by bartektrybala** [reply]
Thx for update
